app/assets/stylesheets/simple_form_customization.scss

```diff
@@ -1,14 +1,17 @@
 //# hacks for simple-form + bulma
 
-.is-inline-flex {
-  display: inline-flex;
-}
-.select {
-  vertical-align: baseline;
-}
-.form-actions {
-  padding-top: 3em;
-}
-.select:not(.is-multiple):not(.is-loading)::after, .navbar-link:not(.is-arrowless)::after {
-  margin-top: 2.5em;
+.simple-form {
```

Comment:
Only change was to wrap everything here under `.simple-form`. All the other diff lines are just indenting.